src/components/dropdownSearch/examples/DropdownSearchExamples.tsx

```diff
@@ -1,20 +1,53 @@
 import * as React from 'react';
 import * as _ from 'underscore';
+
+import {IDispatch, ReduxConnect} from '../../../utils/ReduxUtils';
 import {UUID} from '../../../utils/UUID';
+import {IDropdownOption} from '../DropdownSearch';
+import {updateOptionsDropdownSearch} from '../DropdownSearchActions';
 import {DropdownSearchConnected} from '../DropdownSearchConnected';
 import {MultiSelectDropdownSearchConnected} from '../MultiSelectDropdownSearch/MultiSelectDropdownSearchConnected';
 
-export class DropdownSearchExamples extends React.Component<any, any> {
+interface DropdownSearchExamplesProps {
+    onOptionsChanged?: (id: string, options: IDropdownOption[]) => void;
+}
+
+const mapStateToProps = () => ({});
+
+const mapDispatchToProps = (dispatch: IDispatch): DropdownSearchExamplesProps => ({
+    onOptionsChanged: (id: string, options: IDropdownOption[]) => dispatch(updateOptionsDropdownSearch(id, options)),
+});
+
+@ReduxConnect(mapStateToProps, mapDispatchToProps)

```

Comment:
this has been outdated https://github.com/coveo/react-vapor/pull/668#discussion_r193418478

Reply:
I've seen it anyway. I make sure to read all comments, even if they are hidden.

Reply:
good to know 👍